lazyflow/operators/ioOperators/opStreamingH5N5Reader.py

```diff
@@ -22,21 +22,68 @@
 # Python
 import logging
 import time
-import numpy
+from typing import Union
+
 import vigra
 import h5py
 import z5py
-import json
 import os
-import numpy as np
 
 from lazyflow.graph import Operator, InputSlot, OutputSlot
 from lazyflow.utility import Timer
 from lazyflow.utility.helpers import get_default_axisordering, bigintprod
+from lazyflow.utility.io_util.OMEZarrStore import get_axistags_from_spec as get_ome_zarr_axistags
 
 logger = logging.getLogger(__name__)
 
 
+def _find_or_infer_axistags(file: Union[h5py.File, z5py.N5File, z5py.ZarrFile], internalPath: str) -> vigra.AxisTags:
```

Comment:
great to have single clear place for this. In principle one could differentiate by file type, and not try h5/n5 files for ome metadata, and vice versa...

Reply:
It is a bit weird that this makes OME-H5 and OME-N5 a practical reality 🙃

It would probably be sensible to put the OME checking behind a ZarrFile condition and the N5 check behind an N5File condition. But I would keep the preference for vigra AxisTags common to all.